image/oci/manifest.py

```diff
@@ -129,6 +136,15 @@ def get_meta_schema(self, ignore_unknown_mediatypes=False):
                         ignore_unknown_mediatypes=ignore_unknown_mediatypes,
                     ),
                 },
+                OCI_MANIFEST_SUBJECT_KEY: get_descriptor_schema(
+                    [],
+                    ignore_unknown_mediatypes=True,
+                ),
+                OCI_MANIFEST_ANNOTATIONS_KEY: {
+                    "type": "object",
+                    "description": "The annotations, if any, on this manifest",
+                    "additionalProperties": True,
+                },
```

Comment:
According to [the spec](https://github.com/opencontainers/image-spec/blob/main/annotations.md#rules), there are rules how annotations can be added. We should probably add those validations as well at some point.

Reply:
It would be nice to have at some point. But for this, I don't think the all the constraints in the spec can be enforced using only jsonschema.